lib/util/http/index.ts

```diff
@@ -55,6 +55,7 @@ export class Http<GetOptions = HttpOptions, PostOptions = HttpPostOptions> {
     // TODO: deep merge in order to merge headers
     let options: any = {
       method: 'get',
+      retry: 0,
```

Comment:
Btw do we need `retry=0` for streams? @viceice

Reply:
According to tests, it doesn't do retry though

Reply:
We *do* want `got` to do retrying. Does this disable it?

Reply:
It disables it by default, in order to match current snapshots. Probably, we should adjust snapshots instead.

Reply:
Let's revert it then and make whatever changes necessary to get tests to pas s